src/components/calendar/Calendar.js

```diff
@@ -1,15 +1,293 @@
+                    // This block takes care of the empty cells after last day of month
+                    // Do NOT increment dayCount
+                    rowCells.push(<div className="cell"></div>);
+                } else if (i > 0) {
+                    rowCells.push(
+                        <div className={"cell" + (this.isCurrentDate(dayCount) ? ' today' : '')}>
+                            <div className="day-number">{dayCount}</div>
+                            {this.getEvents(dayCount)}
+                        </div>);
+                    dayCount++;
+                } else {
+                    // This block takes care of the empty cells before the 1st of month
+                    // Do NOT increment dayCount
+                    rowCells.push(<div className="cell"></div>);
+                }
+
+            }
+            // rowCells will be created inside inner for-loop
+            calendar.push(<div className={"row-" + i}>{rowCells}</div>);
+        }   
+
+        return(
+            <div className="calendar-container">
+
+                <div className="calendar-header">
+                    <button className="prev-button" onClick={this.decrementMonth}>Prev</button>
+                    <button className="next-button" onClick={this.incrementMonth}>Next</button>
+                    
+                    <div className="month-header" style={{color: 'white'}}>
+                        {months[monthIndex]} {this.state.currentDate.getFullYear()}
+                    </div>
+                </div>
+
+                <div className="day-header-container">
+                    <div className="day-header">
+                        <span className="test1">Sunday</span>
+                        <span className="test2">Sun</span>
+                    </div>
+                    <div className="day-header">
+                        <span className="test1">Monday</span>
+                        <span className="test2">Mon</span>
+                    </div>
+                    <div className="day-header">
+                        <span className="test1">Tuesday</span>
+                        <span className="test2">Tue</span>
+                    </div>
+                    <div className="day-header">
+                        <span className="test1">Wednesday</span>
+                        <span className="test2">Wed</span>
+                    </div>
+                    <div className="day-header">
+                        <span className="test1">Thursday</span>
+                        <span className="test2">Thu</span>
+                    </div>
+                    <div className="day-header">
+                        <span className="test1">Friday</span>
+                        <span className="test2">Fri</span>
+                    </div>
+                    <div className="day-header">
+                        <span className="test1">Saturday</span>
+                        <span className="test2">Sat</span>
```

Comment:
For the future: better class names (more descriptive), but I'm sure you just forgot to change this when you were finished. I'll have this as a task in the future to clean the css (which I'll probably do).